fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkWriter.java

```diff
@@ -143,6 +156,9 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
             for (Resource fhirResource : fhirResourceList) {
```

Comment:
line 141, please change objResJasonList to objResJsonList

Reply:
ha, good catch, typical typo error! maybe I did thought about Jason at that time point. :)

Reply:
done